nuspec/nuget/Cake.Http.nuspec

```diff
@@ -17,8 +17,14 @@
       <tags>cake script build httpclient web</tags>
    </metadata>
    <files>
-      <file src="Cake.Http.dll" target="lib/net45" />
-      <file src="Cake.Http.pdb" target="lib/net45" />
-      <file src="Cake.Http.xml" target="lib/net45" />
+      <file src="net45/Cake.Http.dll" target="lib/net45" />
```

Comment:
@phenixdotnet If you are going to continue using this nuspec file, then the setting for ShouldRunDotNetPack should be set to false in the setup.cake file, so that there isn't duplication of effort.